mathics/builtin/fileformats/htmlformat.py

```diff
@@ -20,11 +20,15 @@
 import re
 import sys
 
+
 try:
     import lxml.html as lhtml
+
+    lhtml_available = True
 except ImportError:
-    print("lxml.html is not available...")
-    pass
+    from mathics.builtin.fileformats.xmlformat import  parse_xml_stream
```

Comment:
I tried using this on the HTML import example and parse_xml_stream wasn't suitable replacement as far as I could tell.